### DISCUSSION TITLE
How to prevent automatic closing all open tree nodes

### DISCUSSION THREAD

**Discussion by ShparagaVladimir** [ant-design-blazor/ant-design-blazor#2055] [Q&A]
![GqB0ovReN9](https://user-images.githubusercontent.com/56558503/138459837-61eaa349-1cb0-4ec0-be6d-188613157726.gif)

**Comment by anddrzejb** [comment]
Can you confirm you have the same issue as in #2037?

**Reply by ShparagaVladimir** [reply]
The behavior is similar, but I haven't used OnClick. I found what the problem is, by default the search field is empty and even the call to the ExpandAll method does not work, but if you enter any value other than null or "" then the method expands the entire structure and the tree does not hide the nodes when you click on any of them. I think there should be a way to turn off this behavior, if not, I would like to see such a feature added.
